### PR TITLE
Assignment-1 review

### PR DESCRIPTION
**Pull request by adyashree-mahapatra**

### CODE REVIEW

File: Assignment-1/app.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if request.method == "POST":
+        dir = request.form.get("fname")
+        results=list_files(dir)
+    return render_template("index.html",results=results,folder=dir)  
+
+# Function to get the filenames in the folders of AWS Bucket
+def list_files(dir):
+    # AWS Credentials
+    AWS_STORAGE_BUCKET_NAME='1668-mybucket'
+    AWS_S3_REGION_NAME='ap-south-1'
```
Comment (leninkumar-sv-tiger) [inline Assignment-1/app.py:27]: Please refer regions from ENV variables.

File: Assignment-1/app.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    dir = ""
+    if request.method == "POST":
+        dir = request.form.get("fname")
+        results=list_files(dir)
+    return render_template("index.html",results=results,folder=dir)  
+
+# Function to get the filenames in the folders of AWS Bucket
+def list_files(dir):
+    # AWS Credentials
+    AWS_STORAGE_BUCKET_NAME='1668-mybucket'
```
Comment (leninkumar-sv-tiger) [inline Assignment-1/app.py:26]: Please use boto3 to list buckets and select from the list

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
-# AWS_Sagemaker_Assignments
\ No newline at end of file
+# **Assignment-1**
```
Comment (leninkumar-sv-tiger) [inline README.md:1]: Please paste image containing daily cost incurred grouped by Service and Tag

**Review by leninkumar-sv-tiger — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (adyashree-mahapatra) [reply README.md:1]: I have pasted the images of daily cost incurred group by service and tags in the README.md.

File: Assignment-1/app.py (hunk 1 above)
Comment (adyashree-mahapatra) [reply Assignment-1/app.py]: Removed the region variable mentioned in the code

File: Assignment-1/app.py (hunk 2 above)
Comment (adyashree-mahapatra) [reply Assignment-1/app.py]: I have updated the code to take bucket name from the list generated using boto3.

**Review by leninkumar-sv-tiger — APPROVED** [review]
Thanks for making the changes. Please merge to main

**Pull request merged**